runelite-client/src/main/java/net/runelite/client/plugins/notes/NotesPlugin.java

```diff
@@ -77,4 +79,14 @@ protected void shutDown()
 	{
 		ui.getPluginToolbar().removeNavigation(navButton);
 	}
+
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (event.getGroup().equals("notes") && event.getKey().equals("notesData"))
+		{
+			// updates notes with new config value
+			panel.setNotes(event.getNewValue());
```

Comment:
we usually to prefer calling directly the config option here (as when this event occurs, the config object is already updated.

Reply:
Should this be a SessionOpen event, then? I was using ConfigChanged so I had the value in the event.

Reply:
I think it should be SessionOpen, it is called after the config has been applied so you can get the note from the config.